mods/seasonal/moves.js

```diff
@@ -2611,7 +2611,7 @@ let BattleMovedex = {
 					}
 				}
 				if (!positiveBoost || !target.lastMove) return;
-				target.deductPP(target.lastMove.id, target.lastMove.pp);
+				target.lastMove.pp = 0;
```

Comment:
Does this show up on client for the pokemon that lost PP?

Reply:
haven’t been able to test.